crypto/evp/e_aes.c

```diff
@@ -3416,6 +3444,24 @@ static int aes_xts_init_key(EVP_CIPHER_CTX *ctx, const unsigned char *key,
 
     if (key)
         do {
```

Comment:
Indent the body of the do/while.  Consider putting braces on the if body, for clarity.